### ISSUE TITLE
Blue Sky :(

### ISSUE THREAD

**Issue by Jaw1985** [Arkensor/DayZCommunityOfflineMode#20]
Hey, thanks for the great work! All I seem to get is a blue sky when I install your files. I've even reinstalled the game, same issue. Any help appreciated.

**Comment by Jaw1985** [comment]
Ignore me, wasn't installing correctly.

**Comment by blackwolfmagic** [comment]
How did you solve this?

**Comment by Jaw1985** [comment]
I wasn’t placing the mission folder in correctly, only the contents. In this instance you need to replace the entire folder within the missions folder with this one.

**Comment by blackwolfmagic** [comment]
Yeah, that will do it. Here was my problem:

I had the blue screen issue with the unofficial mod. Here's how I solved it. So, my steam was installed under C:\Steam. I also had D:\ which was a separate SDD. DayZ was installed at my separate SDD at D:\DayZ which I installed there using the Steam option "Move Install Folder". No matter what, I couldn't get the mod to work. I had to uninstall DayZ and reinstall at D:\Steam\steamapps\common\DayZ - Finally after doing this, unofficial mode worked.
